filament/src/fg/FrameGraph.cpp

```diff
@@ -455,11 +455,22 @@ FrameGraph& FrameGraph::compile() noexcept {
             //  attachment would be missing).
             for (fg::RenderTarget& rt : renderTargets) {
                 auto& textures = rt.desc.attachments.textures;
-                if (textures[0].isValid()) {
-                    FrameGraphHandle handle = textures[0];
+                FrameGraphHandle handle = textures[0];
+                if (handle.isValid()) {
                     ResourceNode const& node = resourceNodes[handle.index];
                     if (node.resource->imported && node.resource == from.resource) {
                         for (size_t i = 1; i < textures.size(); ++i) {
+                            // FIXME: here we're likely leaking the resources we're clearing,
```

Comment:
File a bug?

Reply:
The "leak" is only during the frame, it's really not that bad --  I track those small issues with FIXMEs.